app/views/mechanics/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<ul id="single_mechanic">
+  <%= "Mechanic: #{@mechanic.name}" %></br>
+  <%= "Years of Experience: #{@mechanic.years_experience}" %> </br>
+
+  <h2>Current rides they're working on:</h2>
+  <% @mechanic.rides.each do |ride| %>
+    <td><%= ride.name %></td>
+  <% end %></br>
+
+  <h2>Add a ride to workload:</h2>
+  <%= form_with url: "/mechanics/#{@mechanic.id}", method: :post, data: { turbo: false } do |f| %>
+    Ride Id: <%= f.text_field :ride_id %><br>
+  <%= f.submit "Submit"%>
+  <% end %>
+</ul>
```

Comment:
Following through on that idea of moving your `create` action to another controller. You would run into an issue of passing that `mechanic_id` through with your request. You can take a look into adding hidden fields in forms. That should allow you to still pass that mechanic id, and route to something like `POST /mechanic_rides`